documentation/reference/rank-features.html

```diff
@@ -750,11 +750,29 @@ <h2 id="feature-list">Feature list</h2>
 <tr><td colspan="3" class="divider" style="background-color: lightgrey;">
 <strong>Rank scores</strong></td></tr>
 
+<tr id="bm25"><td>bm25(field)</td>
+  <td>0</td>
+  <td>
+    Calculates the
+    <a href="https://en.wikipedia.org/wiki/Okapi_BM25">Okapi BM25</a>
+    ranking function over the given
+    <a href="search-definitions-reference.html#indexing-index">indexed string field</a>.
+    This feature is very cheap to compute, about 3-4 times faster than nativeRank,
+    while still providing a good rank score quality wise.
+    This feature is a good candidate for usage in a first phase ranking function when ranking text documents.
+    Note that the field must be enabled to be used with the bm25 feature,
+    set the <em>enable-bm25</em> flag in the
+    <a href="search-definitions-reference.html#index">index</a>
+    section of the field definition.
+    See the <a href="bm25.html">BM25 Reference</a> for more detailed information.
+  </td></tr>
```

Comment:
Please add something about bm25 rank feature not (yet) being supported for streaming search.

Reply:
Fixed